cc/concurrent_selfplay.cc

```diff
@@ -64,14 +64,9 @@
 //    an output queue for `OutputThread` to consume.
 
 // Inference flags.
-DEFINE_string(engine, "tf",
-              "Name of the inference engine to use, e.g. \"tf\", \"tpu\", "
-              "\"lite\"");
 DEFINE_string(device, "",
-              "ID of the device to run inference on. Can be left empty for "
-              "single GPU machines. For a machine with N GPUs, a device ID "
-              "should be specified in the range [0, N). For TPUs, pass the "
-              "gRPC address for the device ID.");
+              "Optional ID of the device to run inference on. For TPUs, pass "
```

Comment:
Do GPUs use the device id or is that entirely handled with CUDA_VISIBLE_DEVICES?